frontend/src/types/LogRecordTypes.ts

```diff
@@ -1,8 +1,8 @@
 export type LogRecord = {
   logId: number;
-  attnTo: number;
-  attnToFirstName: string;
-  attnToLastName: string;
+  attnTo?: number;
+  attnToFirstName?: string;
+  attnToLastName?: string;
```

Comment:
@connor-bechthold I recall you mentioned there was an issue with creating log records because of the `attnTo` flag ; does this fix it?